deps/build.jl

```diff
@@ -1,3 +1 @@
-using TimeZones: build
-
-build()
+using TimeZones
```

Comment:
Hmmm, if someone manually runs `] build` we should probably re-build the scratch directory.